static/js/biomedical/disease/page.tsx

```diff
@@ -87,7 +97,15 @@ export class Page extends React.Component<PagePropType, PageStateType> {
       { id: "name", name: "Compound Name" },
       { id: "drugSource", name: "Drug Source" },
     ];
-
+    const diseasePrevalenceStatVarValue =
+      "Count_MedicalConditionIncident_Condition" + this.props.dcid;
+    const statVarDisease = {} as StatVarSpec;
+    statVarDisease.statVar = diseasePrevalenceStatVarValue;
+    statVarDisease.denom = "";
```

Comment:
Since we are showing "prevalence", should this be Count_Person? That might also negate the changes in map_tile when there is no denominator defined.

Reply:
+1 - showing just the count of a disease might not be super meaningful because that count can be impacted by the population of a certain area

Reply:
Displaying percentages instead of counts! Thanks for the great suggestion!